example_kubeflow_pipelines/openvino/container/Dockerfile

```diff
@@ -0,0 +1,162 @@
+FROM intelpython/intelpython3_core as TENSORFLOW
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        openjdk-8-jdk \
+        openjdk-8-jre-headless \
+        build-essential \
+        curl \
+        git \
+        libcurl3-dev \
+        libfreetype6-dev \
+        libhdf5-serial-dev \
+        libpng-dev \
+        libzmq3-dev \
+        pkg-config \
+        rsync \
+        software-properties-common \
+        unzip \
+        zip \
+        zlib1g-dev && \
+        apt-get clean
+
+RUN git clone -b r1.14 --depth 1 https://github.com/tensorflow/tensorflow
```

Comment:
do you want to make this as arg with default value?
```ARG TENSORFLOW_VERSION= r1.14```

Reply:
I'll keep it in mind for next version